assets/scripts/lobby/sockets/myNotifications.js

```diff
@@ -10,8 +10,12 @@ socket.on("success-alert", function (data) {
     showSuccessAlert(data);
 });
 
-socket.on("danger-alert", function (data) {
-    showDangerAlert(data);
+socket.on("danger-alert", function (data, action = "show") {
```

Comment:
Nice default parameter! I haven't seen that before until now :). Learnt something new!

Reply:
You're welcome. :)